app/src/main/java/org/stacks/app/ui/account/IdentityRowView.kt

```diff
@@ -0,0 +1,32 @@
+package org.stacks.app.ui.account
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.FrameLayout
+import kotlinx.android.synthetic.main.partial_identity_row.view.*
+import org.stacks.app.R
+import org.stacks.app.data.IdentityModel
+
+@SuppressLint("ViewConstructor")
+class IdentityRowView
+@JvmOverloads constructor(
+    context: Context,
+    identityModel: IdentityModel,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    defStyleRes: Int = 0
```

Comment:
No need to add this here. If you pass another value in the constructor like this, you no longer can use the view in XML.